tests/TestIddict.elm

```diff
@@ -273,3 +273,16 @@ insert =
                             (Iddict.size d + 1)
             )
         ]
+
+
+fromList : Test
```

Comment:
Can you move this into the same `Test`  variable as the other tests?

Alternatively, the upper level on the `suite` variable might need to be broken into multiple variables for improved consistency.

Reply:
I'm not sure I get what you're asking. Would you prefer me to put this test inside the `insert` tests?

Reply:
No, I've confused this test file with a similar file from a different repository.